src/TropicalGeometry/hypersurface.jl

```diff
@@ -263,3 +263,39 @@ end
 #     @req has_attribute(TropH,:tropical_semiring_map) "no tropical semiring map cached"
 #     return get_attribute(TropH,:tropical_semiring_map)
 # end
+#
+
+
+@doc raw"""
+	tropical_prevariety(A,minOrMax)
+
+Return the tropical prevariety generated by intersecting tropical hypersurfaces obtained from elements of $A$.
+"""
```

Comment:
Thank you for your contribution! A tiny Julia remark: the docstring for `tropical_prevariety` should be directly attached to the `tropical_prevariety` function, not to `to_tropical`, otherwise `?tropical_prevariety` won't show it